docs/dxp/7.x/en/site-building/displaying-content/using-fragments/adding-a-page-fragment.md

```diff
@@ -26,4 +28,5 @@ Page Fragments are organized in *Collections*. Collections differentiate between
 
 ```note::
   You can also create Page Fragments outside of Liferay Portal using your own tooling. See `Developing a Page Fragment with the Fragments Toolkit <../../developer-guide/developing-page-fragments/developing-page-fragments-with-the-fragments-toolkit.md#collection-format>`_ for more information.
-```
\ No newline at end of file
+```
```

Comment:
I kept going back and forth between adding it to the user guide or the developer guide myself. I leaned towards leaving it in the user guide because it wasn't developer heavy and only involved using the UI, but I see what you're saying. I'll port it to the dev guide instead.